### DISCUSSION TITLE
Deno not defined when using pixie.js

### DISCUSSION THREAD

**Discussion by dagda1** [freshframework/fresh#646] [Q&A]
I am trying to use [pixi.js](https://github.com/pixijs/pixijs) from fresh but when I try and use anything imported by `pixijs` in an island component, e.g.

```ts
import { Application } from "pixi.js";


// inside component
const app = new Application({ width, height, antialias: !0, transparent: !0, resolution: 1 });
```

I get the following error message

> Uncaught ReferenceError: Deno is not defined

**Comment by TranquilMarmot** [comment]
Where are you importing it from? Is it in your import map?
